test/system/201-run.bats

```diff
@@ -4,12 +4,12 @@ load helpers
 
 @test "Echo 'Hello World' inside of the default container" {
   run_toolbox run echo "Hello World"
-  is "$output" "Hello World" "Should say 'Hello World'"
+  # is "$output" "Hello World" "Should say 'Hello World'"
```

Comment:
Should we talk to the Zuul folks about this?

I am happy to turn off the assertion as a temporary measure. I just hope that we don't forget about it.

Reply:
A similar issue was brought up by the Zuul folks some time ago already (https://github.com/containers/toolbox/issues/410). I think we can track the issue there (the issue title should be updated, though).

Reply:
Ok, that's great.

Nitpick: we should link to https://github.com/containers/toolbox/issues/410 from this commit to complete the paper trail.